jsonnetfile.jsonnet

```
@@ -21,6 +22,7 @@ local kube_thanos_version_map = {
   '1.20': 'release-0.17',  // corresponds to v0.17.0
   '1.21': 'release-0.19',  // corresponds to v0.19.0
   '1.22': 'release-0.22',  // corresponds to v0.22.0
+  '1.23': 'v0.26.0',
```

K8s 1.23 is OpenShift 4.10, so this should be v0.23.0 (cf. https://github.com/openshift/cluster-monitoring-operator/blob/release-4.10/jsonnet/jsonnetfile.json#L58), and OpenShift 4.11 (1.24) should be v0.26.0:

```suggestion
  '1.23': 'v0.23.0',
  '1.24': 'v0.26.0',
```